src/client/src/rt-components/open-fin/OpenFinBrowserLink.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC, MouseEvent, ReactNode } from 'react'
+
+interface Props {
+  href: string
+  children: ReactNode
+  className?: string
+}
+
+const OpenFinBrowserLink: FC<Props> = props => (
+  <a
+    {...props}
+    onClick={(e: MouseEvent<HTMLAnchorElement>) => {
+      e.preventDefault()
+      fin.System.openUrlWithBrowser(e.currentTarget.href)
```

Comment:
Adding `href` attribute to the anchor isn't necessarily needed, but I chose to use it to get the native anchor tag behaviours for free.

Reply:
the method handling onClick should be placed outside of the function i.e.
onClick={handleClick}

Reply:
Good spot, addressed in a580156